docs/optics.md

```diff
@@ -8,7 +8,7 @@ circe provides support for optics by integrating with [Monocle](https://julien-t
 dependency on `circe-optics` to your build:
 
 ```scala
-libraryDependencies += "io.circe" %% "circe-optics" % circeVersion
+libraryDependencies += "io.circe" %% "circe-optics" % "@VERSION@"
```

Comment:
circe-optics is 0.15.0
https://github.com/circe/circe-optics

Reply:
ohh I forgot it's a separate repo. I'll hardcode then, I guess